skorch/net.py

```diff
@@ -1325,6 +1327,44 @@ def load_params(self, f):
 
         self.module_.load_state_dict(model)
 
+    def save_history(self, f):
+        """Saves the history of ``NeuralNet`` as a json file.
+
+        Parameters
+        ----------
+        f : file-like object or str
+
+        Examples
+        --------
+        >>> before = NeuralNetClassifier(mymodule)
+        >>> before.save_history('path/to/file.json')
+        >>> before.fit(X, y)
+        >>> after = NeuralNetClassifier(mymodule).initialize()
+        >>> after.load_history('path/to/file.json')
+
+        """
+        with _with_file_like(f, 'w') as fp:
+            json.dump(self.history.to_list(), fp)
+
+    def load_history(self, f):
+        """Load the history of a ``NeuralNet`` from a json file.
+
+        Parameters
+        ----------
+        f : file-like object or str
+
+        Examples
+        --------
+        >>> before = NeuralNetClassifier(mymodule)
+        >>> before.save_history('path/to/file.json')
+        >>> before.fit(X, y)
+        >>> after = NeuralNetClassifier(mymodule).initialize()
+        >>> after.load_history('path/to/file.json')
```

Comment:
I don't think copying the example from above is a good idea. Maybe reference the example from `save_history`?